api/v1/submissions/PKPSubmissionController.php

```diff
@@ -1617,6 +1617,21 @@ public function addContributor(Request $illuminateRequest): JsonResponse
             ], Response::HTTP_FORBIDDEN);
         }
 
+        // validate that if userGroupId is passed, it really is an author group and that email is present.
+        $validator = Validator::make($illuminateRequest->all(), [
+            'userGroupId' => [
+                'nullable',
+                'integer',
+                Rule::exists('user_groups', 'user_group_id')
+                    ->where('role_id', Role::ROLE_ID_AUTHOR)
+                    ->where('context_id', $submission->getData('contextId')),
+            ],
+            'email' => ['required', 'email'],
```

Comment:
do we need to validate the email here as I see we do validate it below when we run `Repo::author()->validate` as in `author.json` , it's validation rule set as `email_or_localhost` .